firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteGlobalsCache.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore.local;
+
+import com.google.protobuf.ByteString;
+
+public class SQLiteGlobalsCache implements GlobalsCache {
+
+    private static final String SESSION_TOKEN = "sessionToken";
+    private final SQLitePersistence db;
+
+    public SQLiteGlobalsCache(SQLitePersistence persistence) {
+        this.db = persistence;
+    }
+
+    @Override
+    public ByteString getSessionsToken() {
```

Comment:
Annotate return value with `@Nullable` please.

Reply:
I changed code to return `ByteString.EMPTY` instead of null.